Hunting Queries/MultipleDataSources/NetworkConnectionldap_log4j.yaml

```
@@ -1,9 +1,9 @@
 id: 19abc034-139e-4e64-a05d-cb07ce8b003b
 name: Malicious Connection to LDAP port for CVE-2021-44228 vulnerability
 description: |
-  'This hunting query looks for connection to LDAP port to find possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability. 
-   Log4j is an open-source Apache logging library that is used in many Java-based applications. Awarness of normal baseline traffic of an enviornment for java.exe
-   while using this query will help detrmine normal from anaomalous.
+  'This hunting query looks for connection to the most common LDAP ports to find possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability.
```

I'd personally change _most common_ with _default_
```suggestion
  'This hunting query looks for connection to the default LDAP ports to find possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability.
```